docs/with-props.md

```diff
@@ -1,26 +1,8 @@
 ---
-title: "Attaching Props"
+title: 'Attaching Props'
 ---
 
-Sometimes it's useful to create components that already have props applied, like the example below with a password input. You use recompose's `withProps` higher-order component to do this.
-
-**[`withProps` documentation](https://github.com/acdlite/recompose/blob/master/docs/API.md#withprops)**
-
-```jsx
-// @live
-import styled from '@emotion/styled'
-import withProps from 'recompose/withProps'
-
-const RedPasswordInput = withProps({
-  type: 'password'
-})(styled('input')`
-  background-color: red;
-`)
-
-render(<RedPasswordInput />)
-```
-
-Alternatively, you can use the css prop and create a regular component like this.
+Some css-in-js libraries offer APIs to attach props to components, instead of having our own API to do that, we recommend creating a regular react component, using the css prop and attaching props like you would with any other React component.
```

Comment:
while this obviously should not require an example, maybe it's worth to add one, just so we can refer people to it when they start asking

Reply:
The example for it is below.